tool/inits/GADGET_hdf5_PAR_IC.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3.9
+import h5py
+import numpy as np
+import math
+
+# "GADGET" input hdf5 file
+input_GADGET_file = "Data_000.hdf5"
```

Comment:
Mention that an example Gadget file with 128^3 DM particles can be downloaded via `curl -JO http://use.yt/upload/1d18ed50`.